lib/plugins/features/movies/controller.js

```diff
@@ -3,8 +3,36 @@
 const Movie = require('../../../models/movie');
 
 exports.create = (payload) => {
-
-  return new Movie().save(payload)
+  const object = { name: payload.title };
+  return new Movie().save(object)
   .then((movie) => new Movie({ id: movie.id }).fetch());
-  
+
 };
+
+exports.findAll = (params) => {
+  console.log('here');
+
+  return new Movie().query( (qb) => {
+    if (params) {
```

Comment:
nit: hopefully params is always defined as an object. If it's not, it's cleaner to put the following at the top of your function:

```
params = params || {};
//or
filter = params || {};
```